document-store/src/main/java/org/hypertrace/core/documentstore/expression/AggregationOperator.java

```diff
@@ -0,0 +1,8 @@
+package org.hypertrace.core.documentstore.expression;
+
+public enum AggregationOperator {
```

Comment:
can we move all the various operators under a package of its own and similarly the interfaces and the impl classes